doc/_static/custom_styles.css

```
@@ -15,14 +15,18 @@ div.fasb h3 {
   display: inline-block;
 }
 
+/* Custom bottom margin for the Where to go now section */
+#where-to-go-now {
+	margin-bottom: 18px;
+}
+
 /* Custom color for FA icons in the Where to go now section */
 #where-to-go-now div.fasb::before {
   color: #36557c;
   width: 32px;
 }
 
 /* Custom colors for specific FA icons in the index page */
-
 div.fa-book::before {
```

```suggestion

div.fa-book::before {
```

This blank line  is here because the comment describes all the selectors beneath it, not just the one immediately below as would be implied by a lack of one.

---

It looks odd with respect to the other comments. Please suggest one that applies specifically to the div below then.

---

Yeah, but then every icon coloring would need a comment, which would get repetitive fast and take up space without really explaining anything that isn't obvious from the first comment. Instead, we can set the comment off as a heading for multiple selectors via formatting like this, so its clear its not supposed to parallel with the others:

`/*** Custom colors for specific FA icons in the index page ***/`

---

Please don't plan for a non-existing future. If more comments are necessary, we'll see what to do then. For now, please suggest a comment that makes sense for the adjustments we have below.

---

I'm not referring to a "non-existing [sic] future", I'm talking about right now. As I've stated previously, the CSS comment describes all three blank-line-separated selector blocks following it, not just the one it is immediately below it as the lack of a blank line below it implies, following the [Google CSS style guide](https://google.github.io/styleguide/htmlcssguide.html) and standard practice (including in the Pandas theme ours is based on).

I've suggested an alternative that tries to address your personal aesthetic opinion about it not "looking odd with respect to the other comments", while keeping this conceptual clarity intact, but if you still don't like the look of it, we can follow the style used by our theme and make _all_ comments section comments with blank lines before and after and not use single-block comments at all, so there is no visual inconsistency between any of them, like this:

<details>

```
/* Ensure FA icons applied to section divs display inline */

div.fasb h3 {
  display: inline-block;
}

/* Custom colors for specific FA icons in the index page */

div.fa-book::before {
  color: #6a3514;
}

div.fa-globe-americas::before {
  color: #1c4489;
}

div.fa-user-friends::before {
  color: #f6951f;
}
```

</details>

---

> /*** Custom colors for specific FA icons in the index page ***/

Changing the comment to that one makes sense to me. Thanks for the suggestion.

---

Sure. I can make note of this in the style guide for the future; I need to update it with a few other things we've agreed on but not formally documented.